wackywebm.js

```diff
@@ -110,8 +116,10 @@ async function main() {
 		lines = [],
 		width = maxWidth,
 		height = maxHeight
+
 	process.stdout.write(`Converting frames to webm (File ${index}/${tempFramesFrames.length})...`)
-	for (const { file } of tempFramesFrames) {
+
+    for (const { file } of tempFramesFrames) {
```

Comment:
Why the change in indent?

Reply:
i was messing with a bunch of stuff for testing and accidentally left it in thats my bad